testsuite/tests/singlecluster/gateway/reconciliation/change_targetref/test_update_authpolicy_target_ref.py

```diff
@@ -0,0 +1,44 @@
+"""
+Test for changing targetRef field in AuthPolicy
+"""
+
+import pytest
+
+from testsuite.kuadrant.policy.authorization.auth_policy import AuthPolicy
+
+pytestmark = [pytest.mark.kuadrant_only, pytest.mark.dnspolicy]
+
+
+@pytest.fixture(scope="module")
+def authorization(
+    oidc_provider, gateway, cluster, blame, module_label, route, route_b
```

Comment:
I think both route fixtures here are already been requested through the client fixtures inside the test